packages/@aws-cdk-testing/framework-integ/test/cloudformation-include/test/integ.novalue-nonstring.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from 'path';
+import * as cdk from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { CfnInclude } from 'aws-cdk-lib/cloudformation-include';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'integ-cfninclude-novalue-boolean');
+
+const templateFile = path.join(__dirname, 'test-templates', 'novalue-boolean.json');
+
+const cfnInclude = new CfnInclude(stack, 'Template', {
+  templateFile,
+});
+
+const bucket = cfnInclude.getResource('TestBucket');
+
+new cdk.CfnOutput(stack, 'BucketName', {
+  value: bucket.ref,
+});
+
+new IntegTest(app, 'CfnIncludeNoValueBooleanTest', {
+  testCases: [stack],
+});
```

Comment:
either add an assertion or a justification for why you don't need to assert here. for future reference mostly